### ISSUE TITLE
list() function should be not verbose.

### ISSUE THREAD

**Issue by bonesoul** [balmasi/migrate-mongoose#10]
```
			return migrator.list()
				.then(migrations => {
```

```
UP:       1462277559915-assign_juice_to_business.js
UP:       1463476921686-user-slugs.js
UP:       1465554485481-simplify-juice-ratings.js
UP:       1466692183680-lowercase_juice_tags.js
UP:       1469097774997-fix-juice-rating-field.js
UP:       1469097774998-fix-hardware-rating-field.js
UP:       1471876808801-calculate-vote-ranks.js
UP:       1472798933688-draft-blog-entries.js
```

im using it programmatically and list should be not printing to console actually :)

**Comment by balmasi** [comment]
Good call! Updating it

**Comment by balmasi** [comment]
Fixed in 2.1.0 👍
